styles/Home.module.css

```diff
@@ -1,47 +1,71 @@
 .main {
   min-height: 100vh;
-  background-color: #EFFAFD;
-
 }
+
 .boxContainer {
-  min-height: 100vH;
+  min-height: 100vh;
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: center;
+  background: url(../img/flipped-ice-aleksandra-khaprenko.jpeg) no-repeat center
```

Comment:
It seem the [unsplash license](https://unsplash.com/license) does not require to put an attribution for this image, but I think we should put it in. Let's do that in another PR maybe?

Reply:
Great idea. That link was provided by the photographer: "Photo by <a href="https://unsplash.com/@xaprrr?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Aleksandra Khaprenko</a> on <a href="https://unsplash.com/?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a>"
  
we can use that later on.